packages/transport/src/chrome-runtime/adapter.ts

```diff
@@ -134,12 +134,7 @@ export const connectChromeRuntimeAdapter = (
         Object.entries(handler) as [keyof UniversalHandler, unknown][],
       ) as { [k in keyof UniversalHandler]: UniversalHandler[k] };
       // wrap the handler function to generate and apply context
-      const wrappedFn: UniversalHandlerFn = req =>
-        opt.createRequestContext(req).then(handlerFn, error => {
-          // it's convenient to log and rethrow as ConnectError here
-          console.warn('Handler Error', handler.name, error);
-          throw ConnectError.from(error);
-        });
+      const wrappedFn: UniversalHandlerFn = req => opt.createRequestContext(req).then(handlerFn);
```

Comment:
if we choose 'do nothing', we should still at least merge this removal of this ineffective rethrow